src/daemon/daemon.cpp

```diff
@@ -1261,7 +1258,18 @@ try // clang-format on
             }
             catch (const mp::SSHFSMissingError&)
             {
-                return grpc_status_for_mount_error(name);
+                try
+                {
+                    MountReply mount_reply;
+                    mount_reply.set_mount_message("Enabling support for mounting");
```

Comment:
This seems like a good reason for the spinner to be restarted in the client.

Reply:
Unless I misunderstand what you are saying here, adding the `spinner.stop()` in mount's `streaming_callback` should do this??

Reply:
yes, this was just me referring back to the same thing